routers/common/repo.go

```diff
@@ -63,6 +63,7 @@ func ServeData(ctx *context.Context, name string, size int64, reader io.Reader)
 	// Google Chrome dislike commas in filenames, so let's change it to a space
 	name = strings.ReplaceAll(name, ",", " ")
 
+	// st, err := typesniffer.DetectContentTypeExtFirst(name, buf)
```

Comment:
Commented code is confusing and we shouldn't have it. If you want to retain it for future reference, it's better to add a sentence or two about it.